raiden/network/rpc/client.py

```diff
@@ -187,8 +187,8 @@ def new_filter(jsonrpc_client, contract_address, topics, from_block=None, to_blo
     if isinstance(to_block, int):
         to_block = hex(to_block)
     json_data = {
-        'fromBlock': from_block if from_block is not None else 'latest',
```

Comment:
If we are keeping the `'earliest'` as the default value, please use the simpler syntax `from_block or 'earliest'` (same thing with latest bellow)